charts/posthog/values.yaml

```diff
@@ -530,6 +530,100 @@ pluginsIngestion:
   sentryDSN:
 
 
+pluginsAnalyticsIngestion:
+  # -- Whether to install the PostHog plugin-server analytics ingestion
+  # capability as an individual workload. Note that this is different from the
+  # `pluginsIngestion` setting above specifically in that the deployment
+  # controlled by these settings does not handle session recordings. This
+  # deployment is intended to be a replacement for `pluginsIngestion` and is
+  # intended to be used along side the `recordingsIngestion` deployment.
+  #
+  # The reason these values were added is to be able to scale analytics and
+  # session recordings events indepentently, and to be able to roll this out in
+  # a backwards compatible way.
+  enabled: false
+
+  # -- Count of plugin-server pods to run. This setting is ignored if
+  # `pluginsAnalyticsIngestion.hpa.enabled` is set to `true`. 
+  replicacount: 1
+
+  hpa:
+    # -- Whether to create a HorizontalPodAutoscaler for the plugin stack.
+    enabled: false
+    # -- CPU threshold percent for the plugin-server stack HorizontalPodAutoscaler.
+    cputhreshold: 60
+    # -- Min pods for the plugin-server stack HorizontalPodAutoscaler.
+    minpods: 1
+    # -- Max pods for the plugin-server stack HorizontalPodAutoscaler.
+    maxpods: 10
+    # -- Set the HPA behavior. See
+    # https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+    # for configuration options
+    behavior:
+
+  rollout:
+    # The max surge in pods during a rollout
+    maxSurge: 25%
+    # The max unavailable during a rollout
+    maxUnavailable: 25%
+
+  # -- Additional env variables to inject into the plugin-server stack deployment.
+  env:
+    - name: INGESTION_OVERFLOW_ENABLED
+      # -- Whether or not to send events to a "slow-lane" to be consumed
+      # separately by the pluginsIngestionOverflow deployment. When setting this
+      # to true, you must set pluginsIngestionOverflow.enabled to `true``
+      # otherwise some events will not be processed in cases of overflow. This
+      # behaviour is triggered for instance if there are too many events to be
+      # processed in near real time from a single device.
+      value: "false"
```

Comment:
I thought this was enabled? 🤔

Reply:
It's enabled in the infra repo [here](https://github.com/PostHog/posthog-cloud-infra/blob/main/helm/values/prod.yml#L230)

Reply:
Perhaps we should enable be default at some point 🤔  what do you think?

Reply:
Ah, okay, I keep getting confused between the two repos. Thank you.

Yeah, I'd say once it has been running for a while we can enable it by default.